data/mods/Aftershock/items/weapons.json

```diff
@@ -90,7 +90,7 @@
     "price": 1000000,
     "price_postapoc": 20000,
     "material": [ "superalloy", "hardsteel" ],
-    "flags": [ "FIRE_TWOHAND", "STR_DRAW", "SLOW_WIELD" ],
+    "flags": [ "FIRE_TWOHAND", "STR_DRAW", "SLOW_WIELD", "WATERPROOF_GUN", "NEVER_JAMS", "WATERPROOF_GUN", "NEVER_JAMS" ],
```

Comment:
Doubled flags waterproof_gun and never_jams

Reply:
good catch, fixed.